specification/datamigration/resource-manager/readme.md

```diff
@@ -22,15 +23,61 @@ To see additional help and options, run:
 ## Configuration
 
 ### Basic Information
+
 These are the global settings for the Database Migration Service API.
 
 ``` yaml
 title: DataMigrationManagementClient
 description: Data Migration Client
 openapi-type: arm
-tag: package-2018-07-15-preview
+tag: package-2021-06
```

Comment:
Should this be package-2021-06-30?

Reply:
I'm not sure - this was generated using the Hub. Whenever I perform the build locally, it correctly loads the 2021-06-30 API version.

Reply:
Okay, as long as its working fine as before it used the full date.